cassovary-core/src/main/scala/com/twitter/cassovary/graph/node/SharedArrayBasedDirectedNode.scala

```diff
@@ -36,7 +36,7 @@ object SharedArrayBasedDirectedNode {
     dir match {
       case StoredGraphDir.OnlyIn | StoredGraphDir.OnlyOut | StoredGraphDir.Mutual =>
         val outEdges: Seq[Int] = Option(edges(nodeId)).getOrElse(emptyArray)
-        UniDirectionalNode(nodeId, outEdges, dir)
+        UniDirectionalNode.applySeq(nodeId, outEdges, dir)
```

Comment:
We should have here outEdges: Array[Int] as well. But this id something for next commits. And let's wait for it until I finish my collections refactoring.